### DISCUSSION TITLE
Support for appended attributes

### DISCUSSION THREAD

**Discussion by plegenza** [rappasoft/laravel-livewire-tables#501] [Feature Requests]
Maybe make support for append attributes.
Example attribute like fullname for model which is concatination firstname and lastname.

Currently sorting on fullname attribute will cause error "Column not found: 1054 Unknown column 'fullname' in 'order clause'".

It can be handled by collection sort for elements which exist in $append array for Model.

**Comment by rappasoft** [comment]
Feel free to make a PR and I'll merge it if it works.

**Reply by jornatf** [reply]
is there a solution for this today?

**Reply by lrljoe** [reply]
**Depending on your use case**, you could utilise the setAdditionalSelects to generate the relevant field for use in the table.
